src/els_methods.erl

```diff
@@ -100,16 +103,16 @@ initialize(Params, State) ->
   ok = els_provider:initialize(),
   Result =
     #{ capabilities =>
-         #{ hoverProvider => els_hover_provider:is_enabled()
-          , completionProvider =>
-              #{ resolveProvider => false
-               , triggerCharacters => [<<":">>, <<"#">>, <<"?">>]
-               }
-          , textDocumentSync =>
+         #{ textDocumentSync =>
```

Comment:
Any specific reason behind this change?

Reply:
I re-ordered the capabilities to line up with the order in the spec. It is easier to see what is needed, and what is extra that way.  The re-ordering could arguably be done in a separate PR.

Reply:
It's fine. I was just curious, since I couldn't see the reason.